ci-operator/config/openshift/metallb/openshift-metallb-main.yaml

```diff
@@ -42,6 +46,26 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: metallb-l2-e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN
```

Comment:
Can we create 2 copies of the L2 job - one that tests with `OpenShiftSDN` and the other with `OVNKubernetes`?  I would also update the job names to indicate which is being used.

Reply:
Sure. Changed.